puppet/modules/jenkins_job_builder/files/theforeman.org/pipelines/lib/foreman.groovy

```diff
@@ -0,0 +1,62 @@
+def databaseFile (id) {
+    writeFile(file: 'config/database.yml', text: """
+test:
+  adapter: postgresql
+  database: ${id}-test
+  username: foreman
+  password: foreman
+  host: localhost
+  template: template0
+
+development:
+  adapter: postgresql
+  database: ${id}-development
+  username: foreman
+  password: foreman
+  host: localhost
+  template: template0
+
+production:
+  adapter: postgresql
+  database: ${id}-development
+  username: foreman
+  password: foreman
+  host: localhost
+  template: template0
+""")
+
+}
+
+def addGem() {
+    sh 'echo "gemspec :path => \'../\', :development_group => :dev" >> bundler.d/Gemfile.local.rb'
+}
+
+def addSettings(settings) {
+    sh "cp config/settings.yaml.example config/settings.yaml"
+
+    if (settings['locations']) {
+        sh "sed -i 's/:locations_enabled: false/:locations_enabled: true/' config/settings.yaml"
+    }
+
+    if (settings['organizations']) {
+        sh "sed -i 's/:organizations_enabled: false/:organizations_enabled: true/' config/settings.yaml"
+    }
+}
+
+def configureDatabase(ruby) {
+    withRVM(['bundle update'], ruby)
+    withRVM(['bundle exec rake db:create -q'], ruby)
```

Comment:
can we *try* to drop the db before creating it? like it was done in https://github.com/theforeman/foreman-infra/pull/436